bodygroupmanager/sv_hooks.lua

```diff
@@ -0,0 +1,7 @@
+local PLUGIN = PLUGIN
+
+netstream.Hook("ixBodygroupTableSet", function(client, target, bodygroups)

```

Comment:
You should do some checks here, otherwise anyone can send a net message to change their bodygroups at any time regardless of access